source/request-as-event-emitter.js

```diff
@@ -122,8 +120,9 @@ module.exports = (options = {}) => {
 
 		cacheReq.once('request', req => {
 			let aborted = false;
-			req.once('abort', _ => {
+			req.once('abort', () => {
 				aborted = true;
+				emitter.emit('abort');
```

Comment:
Do we really need that? It's aborted only when receives an error/someone cancels the request. The error `got` throws has nice details though.

Reply:
It's not needed. I don't know why I put it there. 🤪

Reply:
I don't know.